ros/src/computing/planning/motion/packages/waypoint_follower/include/mpc_follower/mpc_follower.h

```diff
@@ -0,0 +1,234 @@
+  ros::Publisher pub_steer_vel_ctrl_cmd_; //!< @brief topic publisher for control command
+  ros::Publisher pub_twist_cmd_;          //!< @brief topic publisher for twist command
+  ros::Subscriber sub_ref_path_;          //!< @brief topic subscriber for reference waypoints
+  ros::Subscriber sub_pose_;              //!< @brief subscriber for current pose
+  ros::Subscriber sub_vehicle_status_;    //!< @brief subscriber for currrent vehicle status
+  ros::Timer timer_control_;              //!< @brief timer for control command computation
+
+  MPCTrajectory ref_traj_;                                   //!< @brief reference trajectory to be followed
+  Butterworth2dFilter lpf_steering_cmd_;                     //!< @brief lowpass filter for steering command
+  Butterworth2dFilter lpf_lateral_error_;                    //!< @brief lowpass filter for lateral error to calculate derivatie
+  Butterworth2dFilter lpf_yaw_error_;                        //!< @brief lowpass filter for heading error to calculate derivatie
+  autoware_msgs::Lane current_waypoints_;                    //!< @brief current waypoints to be followed
+  std::shared_ptr<VehicleModelInterface> vehicle_model_ptr_; //!< @brief vehicle model for MPC
+  std::string vehicle_model_type_;                           //!< @brief vehicle model type for MPC
+  std::shared_ptr<QPSolverInterface> qpsolver_ptr_;          //!< @brief qp solver for MPC
+  std::string output_interface_;                             //!< @brief output command type
+
+  /* parameters for control*/
+  double ctrl_period_;              //!< @brief control frequency [s]
+  double steering_lpf_cutoff_hz_;   //!< @brief cutoff frequency of lowpass filter for steering command [Hz]
+  double admisible_position_error_; //!< @brief stop MPC calculation when lateral error is large than this value [m]
+  double admisible_yaw_error_deg_;  //!< @brief stop MPC calculation when heading error is large than this value [deg]
+  double steer_lim_deg_;            //!< @brief steering command limit [rad]
+  double wheelbase_;                //!< @brief vehicle wheelbase length [m] to convert steering angle to angular velocity
+  double steering_gear_ratio_;      //!< @brief steering gear ratio to convert steering to steering_wheel
+
+  /* parameters for path smoothing */
+  bool enable_path_smoothing_;     //< @brief flag for path smoothing
+  bool enable_yaw_recalculation_;  //< @brief flag for recalculation of yaw angle after resampling
+  int path_filter_moving_ave_num_; //< @brief param of moving average filter for path smoothing
+  int path_smoothing_times_;       //< @brief number of times of applying path smoothing filter
+  int curvature_smoothing_num_;    //< @brief point-to-point index distance used in curvature calculation
+  double traj_resample_dist_;      //< @brief path resampling interval [m]
+
+  struct MPCParam
+  {
+    int prediction_holizon;                         //< @brief prediction horizon step
```

Comment:
Typo, `prediction_horizon`. You can do find and replace for all ocurrences

Reply:
fix for all codes